curiouslearning/FeedTheMonsterJS
# fix: feedback title text audio issue (FM-341)

# Changes
- Fix syncing feedback text and audio and issue with the level end screen when you don't feed the monster

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level and play
- Make sure feedback title plays in sync with the feedback audio
- Also make sure if you don't feed the monster and run out of time it should go instantly in levelend scene

Ref: [FM-341](https://curiouslearning.atlassian.net/browse/FM-341)


[FM-341]: https://curiouslearning.atlassian.net/browse/FM-341?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
